Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -652,6 +652,15 @@ void WebChromeClient::requestPointerUnlock()
     protectedPage()->send(Messages::WebPageProxy::RequestPointerUnlock());
 }
 
+bool WebChromeClient::pointerLockSupportsUnadjustedMovement() const
```

Comment:
ChromeClient isn't a great place for this because it isn't actually getting information from the client.  It could just be a static function instead that returns different values based on HAVE(MOUSE_UNACCELERATED_MOVEMENT)

Reply:
Moved to a static method on PointerLockController: `supportsUnadjustedMovement`